Backend/dev/migrations-read-only/rider-app/payout_order.sql

```diff
@@ -47,3 +47,9 @@ ALTER TABLE atlas_app.payout_order ADD COLUMN retried_order_id text ;
 ------- SQL updates -------
 
 ALTER TABLE atlas_app.payout_order ADD COLUMN merchant_operating_city_id TYPE text ;
+
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_app.payout_order ALTER COLUMN merchant_operating_city_id TYPE text;
```

Comment:
should be only one migration:
`ALTER TABLE atlas_app.payout_order ALTER COLUMN merchant_operating_city_id text;`